product_docs/docs/biganimal/release/administering_cluster/customizing_compliance/aws_controls.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Remediating AWS control findings"
+
+---
+
+This page describes the control findings in your AWS Security Hub and suggests potential remediation steps for your BigAnimal resources. 
+
+!!! Important
+    Remediation steps provided below might increase your cloud costs. Before proceeding, make sure that you evaluate and prioritize the suggested actions based on your organization policies. 
+
+The control findings are listed below based on the level of impact to your resources. 
```

Comment:
Same remark as above on the usage of "control". Suggested rewording "The findings...".

Reply:
incorporated your feedback